scripts/Imaging/imgpy/docs/filters.md

```diff
@@ -0,0 +1,135 @@
+                        For best results it should avoid being blocked by any object.
+                        Example: --air-region='[150,234,23,22]'
+```
+## Median Filter
+```
+  --pre-median-size PRE_MEDIAN_SIZE
+                        Size / width of the median filter(pre - processing).
+  --pre-median-mode {reflect,constant,nearest,mirror,wrap}
+                        Default: reflect
+                        Mode of median filter which determines how the array borders are handled.
+```
+## Remove stripes
+```
+  --remove-stripes REMOVE_STRIPES
+                        Methods supported: 'wf' (Wavelet-Fourier).
+```
+
+## Rotation
+```
+  -r ROTATION, --rotation ROTATION
+                        Rotate images by 90 degrees a number of times.
+                        The rotation is clockwise unless a negative number is given which indicates rotation counterclockwise.
+```
+## Clip values after flat/dark normalisation
+```
+  --clip-min CLIP_MIN   Default: 0.0
+                        Clip values after normalisations to remove out of bounds pixel values.
+  --clip-max CLIP_MAX   Default: 1.5
+                        Clip values after normalisations to remove out of bounds pixel values.
+```
+## Cut-off **bright** pixels
+```
+  --cut-off CUT_OFF     Default: None
+                        Cut off values above threshold relative to the max pixels.
+```
+## Remove outliers
+```
+  --pre-outliers PRE_OUTLIERS
+                        Crop bright pixels.
+  --pre-outliers-radius PRE_OUTLIERS_RADIUS
+                        Radius for the median filter to determine the outlier.
+```
+## Rebin
+```
+  --rebin REBIN         Rebin factor by which the images will be rebinned. This could be any positive float number.
+                        If not specified no scaling will be done.
+  --rebin-mode {nearest,lanczos,bilinear,bicubic,cubic}
+                        Default: bilinear
+                        Specify which interpolation mode will be used for the scaling of the image.
+```
+## Gaussian Filter
+```
+  --pre-gaussian-size PRE_GAUSSIAN_SIZE
+                        Apply gaussian filter (2d) on reconstructed volume with the given window size.
+  --pre-gaussian-mode {reflect,constant,nearest,mirror,wrap}
+                        Default: reflect
+                        Mode of gaussian filter which determines how the array borders are handled.(pre processing).
+  --pre-gaussian-order PRE_GAUSSIAN_ORDER
+                        Default: 0
+                        The order of the filter along each axis is given as a sequence of integers, 
+                        or as a single number. An order of 0 corresponds to convolution with a Gaussian kernel.
+                        An order of 1, 2, or 3 corresponds to convolution with the first, second or third derivatives of a Gaussian.
+                        Higher order derivatives are not implemented.
+```
+## Minus Log
+```
+  -log, --pre-minus-log
+                        Default: 0
```

Comment:
You're right, I mistakenly put the default value, and for False that seems to put a 0